deployment/ccip/changeset/globals/ocr3.go

```diff
@@ -52,26 +52,25 @@ var (
 	CommitOCRParamsForEthereum = withOverrides(
 		CommitOCRParams,
 		types.OCRParameters{
-			DeltaRound: 90 * time.Second,
-			DeltaStage: 60 * time.Second,
+			DeltaRound: 15 * time.Second,
```

Comment:
Should we have a different OCR round time for ethereum still? Or should we simplify and just have one set of cfgs?

Reply:
Idea behind separate configs for Ethereum and others was driven by the cost reduction. We don't want to spam like crazy and instead batch more efficiently on Ethereum (wait for more messages to get into the CommitReport). Probably 2 second rounds don't make sense on chain that creates a block every 12 second, no?

Reply:
Since neither commit nor exec produce a report every round (exec every 3 rounds, commit every 2 rounds I think) we can do block time / 3 for exec to get commit round time, and block time / 2 for commit to get commit round time?